common/constants/shared.ts

```diff
@@ -57,10 +57,14 @@ export const observabilityMetricsID = 'observability-metrics';
 export const observabilityMetricsTitle = 'Metrics';
 export const observabilityMetricsPluginOrder = 5092;
 
+export const observabilityTracesNewNavID = 'observability-traces-nav';
+export const observabilityTracesNewNavURL = observabilityTracesNewNavID + '#/traces';
 export const observabilityTracesID = 'observability-traces';
 export const observabilityTracesTitle = 'Traces';
 export const observabilityTracesPluginOrder = 5093;
 
+export const observabilityServicesNewNavID = 'observability-services-nav';
```

Comment:
This is same as `observabilityTracesNewNavID`. We should keep only one.

Reply:
The traces one has -traces- instead of services. Needed difference for the new plugin registration.